.github/workflows/pullrequest.yml

```diff
@@ -9,3 +9,6 @@ on:
 jobs:
   test:
     uses: viamrobotics/viam-typescript-sdk/.github/workflows/test.yml@main
+
+  license_check:
+    uses: viamrobotics/viam-typescript-sdk/.github/workflows/license_finder.yml@main
```

Comment:
this file won't exist until we merge - see run from https://github.com/viamrobotics/viam-typescript-sdk/pull/75/commits/fa99b427fb4470f24f3e75bf81e8970fc7eefe9c for an example of a successful run